src/test/java/org/opentripplanner/graph_builder/module/islandpruning/EscalatorPruningTest.java

```diff
@@ -0,0 +1,71 @@
+package org.opentripplanner.graph_builder.module.islandpruning;
+
+import java.io.File;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.opentripplanner.graph_builder.issue.api.DataImportIssueStore;
+import org.opentripplanner.graph_builder.module.osm.OsmModule;
+import org.opentripplanner.openstreetmap.OsmProvider;
+import org.opentripplanner.routing.graph.Graph;
+import org.opentripplanner.test.support.ResourceLoader;
+import org.opentripplanner.transit.model.framework.Deduplicator;
+import org.opentripplanner.transit.service.StopModel;
+import org.opentripplanner.transit.service.TransitModel;
+
+public class EscalatorPruningTest {
+
+  private static Graph graph;
+
+  @BeforeAll
+  static void setup() {
+    graph =
```

Comment:
Since it's only one test, you don't need the global setup method. Just build the graph in the test itself.